File: sdkwork-backend-sdk-go/types/plus_file_vo.go
package types

// 文件VO类，用于存储文件的元数据信息
type PlusFileVO struct {
	ParentUuid string `json:"parentUuid"`
	ParentMetadata PlusTreeParentMetadata `json:"parentMetadata"`
	Id int `json:"id"`
	ParentId int `json:"parentId"`
	Name string `json:"name"`
	Path string `json:"path"`
	FullPath string `json:"fullPath"`
	ObjectKey string `json:"objectKey"`
	Size int `json:"size"`
	ContentType string `json:"contentType"`
	Extension string `json:"extension"`
	StorageClass string `json:"storageClass"`
	VersionId string `json:"versionId"`
	Resource FileMediaResource `json:"resource"`
	Url string `json:"url"`
	UploadTime string `json:"uploadTime"`
	LastAccessTime string `json:"lastAccessTime"`
	Description string `json:"description"`
	Tags TagsContent `json:"tags"`
	FileType string `json:"fileType"`
	FileCategory string `json:"fileCategory"`
	Permission FilePermission `json:"permission"`
	IsPublic bool `json:"isPublic"`
	PinnedAt string `json:"pinnedAt"`
	Status string `json:"status"`
	Content FileContentObject `json:"content"`
	Etag string `json:"etag"`
}

File: sdkwork-backend-sdk-csharp/Models/PlusFileVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusFileVO
    {
        public string? ParentUuid { get; set; }
        public PlusTreeParentMetadata? ParentMetadata { get; set; }
        public int? Id { get; set; }
        public int? ParentId { get; set; }
        public string? Name { get; set; }
        public string? Path { get; set; }
        public string? FullPath { get; set; }
        public string? ObjectKey { get; set; }
        public int? Size { get; set; }
        public string? ContentType { get; set; }
        public string? Extension { get; set; }
        public string? StorageClass { get; set; }
        public string? VersionId { get; set; }
        public FileMediaResource? Resource { get; set; }
        public string? Url { get; set; }
        public string? UploadTime { get; set; }
        public string? LastAccessTime { get; set; }
        public string? Description { get; set; }
        public TagsContent? Tags { get; set; }
        public string? FileType { get; set; }
        public string? FileCategory { get; set; }
        public FilePermission? Permission { get; set; }
        public bool? IsPublic { get; set; }
        public string? PinnedAt { get; set; }
        public string? Status { get; set; }
        public FileContentObject? Content { get; set; }
        public string? Etag { get; set; }
    }
}
